Library/Homebrew/cmd/update-report.rb

```diff
@@ -132,11 +132,15 @@ def output_update_report
         "git", "-C", HOMEBREW_REPOSITORY, "tag", "--list", "--sort=-version:refname", "*.*"
       ).lines.first.chomp
 
-      if old_tag.blank? || (new_tag == old_tag)
+      Settings.write "latesttag", new_tag if new_tag != old_tag
+
+      if new_tag == old_tag
         puts "Updated Homebrew from #{shorten_revision(initial_revision)} to #{shorten_revision(current_revision)}."
+      elsif old_tag.blank?
+        ohai "Updated Homebrew from (#{shorten_revision(initial_revision)}) " \
```

Comment:
```suggestion
        ohai "Updated Homebrew from #{shorten_revision(initial_revision)} " \
```

feels weird to have the revision in parens but no tag.